logstash-scheduler/src/main/java/org/apache/mesos/logstash/scheduler/TaskInfoBuilder.java

```diff
@@ -29,11 +29,16 @@
 
     public Protos.TaskInfo buildTask(Protos.Offer offer) {
 
+        String executorImage = LogstashConstants.EXECUTOR_IMAGE_NAME_WITH_TAG;
+        if (logstashConfig.getExecutorImage() != null && logstashConfig.getExecutorVersion() != null) {
+            executorImage = logstashConfig.getExecutorImage() + ":" + logstashConfig.getExecutorVersion();
+        }
```

Comment:
If you add defaults to configuration, you can get rid of this whole bit. Including the entries in `LogstashConstants`